### PR TITLE
Scogt 82

### PR DESCRIPTION
**Pull request by folkengine**

### CODE REVIEW

File: config/config.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-window.API_HOST = 'http://localhost:4000'
+window.API_HOST = 'https://data.smartcolumbusos.com'
```
Comment (ManApart) [inline config/config.js]: This looks columbus specific. Is there a reason it's being changed from local host?

File: webpack.config.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ module.exports = (env, argv) => {
   ]
 
   return {
-    watch: true,
+    watch: false,
```
Comment (ManApart) [inline webpack.config.js]: Why turn off the test watch here?

**Comment by ManApart** [comment]
It sounds like this is for an issue (Scogt 82) but I'm not familiar with where you guys keep issues etc. Can you link to the issue (if it's public) or provide some more context for this PR?

File: webpack.config.js (hunk 2 above)
Comment (folkengine) [reply webpack.config.js]: It won't finish building if set to watch.

File: config/config.js (hunk 1 above)
Comment (folkengine) [reply config/config.js]: Woops, I thought I pulled this request. Good call. 
I'll update but pulling anyway.

**Pull request closed without merging**